package.json

```diff
@@ -116,11 +117,17 @@
       "default": true,
       "order": 5
     },
+    "enableDicy": {
+      "description": "Enable the usage of the [DiCy](https://yitzchak.github.io/dicy/) builder instead of `latexmk`.",
+      "type": "boolean",
```

Comment:
@thomasjo Cool. I'll make the change since I don't think it would prevent a shift to DiCy if we decide to go there.

Aside from that I think it should be ready to :ship: